# goal definition
string newFilepath
---
# result definition
string processedFilepath
---
#feedback definition
string feedback 
